src/common/context/forms/index.tsx

```diff
@@ -91,6 +93,23 @@ export const FormsContextProvider: FunctionComponent = ({ children }) => {
     setForms(nextForms);
   };
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const setCurrentForm = (data: any, updatedOwnership?: Ownership): void => {
```

Comment:
Can you refactor `setCurrentFormOwnership` and `setCurrentFormData` (and maybe even `setCurrentFileName`) to use this function? This way we only write the logic for the setting once

Reply:
You can use a curry function to make it really neat.
So for instance setCurrentFormData = (data) => setCurrentForm(data, currentFormOwner).